build.sbt

```diff
@@ -22,8 +22,8 @@ developers := List(
   )
 )
 
-scalaVersion := "2.13.8"
-crossScalaVersions := Seq("2.11.12", "2.12.15", "2.12.16", "2.13.7", "2.13.8")
+scalaVersion := "2.13.9"
+crossScalaVersions := Seq("2.11.12", "2.12.15", "2.12.16", "2.13.7", "2.13.8", "2.13.9")
```

Comment:
I think we have the tradition of keeping two minor versions per major version. Thus I suggest to remove `2.13.7`.

Reply:
Ah. Now I can see @saeltz said we are hoarding 3 versions. That's fine. I must've remembered wrong.

Reply:
I wasn’t sure either. In the CI we had three, in SBT two. We can still clean it up afterwards.